runtime/compiler/CMakeLists.txt

```diff
@@ -115,6 +115,22 @@ if(JITSERVER_SUPPORT)
 	)
 
 	add_custom_target(j9jit_proto DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/net/gen/compile.pb.h ${CMAKE_CURRENT_SOURCE_DIR}/net/gen/compile.pb.cpp)
+
+	if(OPENSSL_BUNDLE_LIB_PATH) # --enable-openssl-bundling
+		set(OPENSSL_ROOT_DIR ${OPENSSL_BUNDLE_LIB_PATH})
+		SET(CMAKE_SHARED_LINKER_FLAGS "-Wl,-rpath,$ORIGIN/.. ${CMAKE_SHARED_LINKER_FLAGS}")
+		message("-- OpenSSL OPENSSL_BUNDLE_LIB_PATH: ${OPENSSL_BUNDLE_LIB_PATH}")
+	elseif(OPENSSL_DIR) # --with-openssl=fetched only
+		set(OPENSSL_ROOT_DIR ${OPENSSL_DIR})
+		message("-- OpenSSL OPENSSL_DIR: ${OPENSSL_DIR}")
+	endif()
+
+	include(FindOpenSSL)
+	find_package(OpenSSL REQUIRED)
+	message("-- OpenSSL include dir: ${OPENSSL_INCLUDE_DIR}")
+	message("-- OpenSSL libssl: ${OPENSSL_SSL_LIBRARY} (version ${OPENSSL_VERSION})")
+	message("-- OpenSSL libcrypto: ${OPENSSL_CRYPTO_LIBRARY} (version ${OPENSSL_VERSION})")
```

Comment:
Beyond debugging, I'm not sure of the value of the uses of  `message()` here.

Reply:
To provide more explicit information on which version of libssl JITServer is linked to.

Reply:
I believe this only repeats information displayed at configuration time: Please remove them.